vignettes/ww-rt.Rmd

```diff
@@ -63,43 +63,85 @@ dist.fec = ern::def_dist_fecal_shedding('sarscov2')
+
+```{r initializing smoothing and Rt params}
+# Initializing scaling factor
+scaling.factor = 1
+
 # Initializing smoothing parameters
 prm.smooth = list(
   align  = 'center', # smoothing alignment
-  method = 'loess', # smoothing method
-  span   = 0.29 # smoothing span (used for loess smoothing only)
+  method = 'loess',  # smoothing method
+  span   = 0.29      # smoothing span (used for loess smoothing only)
 )
 
 # Initialzing Rt settings
 prm.R = list(
-  iter = 20, # number of iterations in Rt ensemble
-  CI = 0.95, # confidence interval
-  window = 10, # Time window for Rt calculations
-  config.EpiEstim = EpiEstim::make_config(seed = 15) # EpiEstim configuration for Rt calculations
+  iter   = 20,   # number of iterations in Rt ensemble
+  CI     = 0.95, # confidence interval
+  window = 10,   # Time window for Rt calculations
+  config.EpiEstim = NULL # EpiEstim configuration for Rt calculations
 )
 ```
 
 ## Estimating $\mathcal{R}_t$
 
-Once the above items are loaded, $\mathcal{R}_t$ can be estimated using `estimate_R_ww()`. In this example, the number of iterations will be set to 20 (the default is 100).
+Once the above inputs and parameters are defined, we estimate $\mathcal{R}_t$ as follows:
 
 ```{r estimate rt}
 r.estim = ern::estimate_R_ww(
   ww.conc        = ww.conc,
   dist.fec       = dist.fec,
   dist.gi        = dist.gi,
+  scaling.factor = scaling.factor,
   prm.smooth     = prm.smooth,
   prm.R = prm.R,
   silent = TRUE
 )
 ```
 
-## Visualizing $R_t$ estimates
+`estimate_R_ww()` returns a list with four elements:
+
+- `ww.conc`: the original input of viral concentration in wastewater over time
+
+- `ww.smooth`: the smoothed wastewater concentration
+
+- `inc`: the daily incidence inferred over time, which is computed by performing a deconvolution of the smoothed wastewater concentration with a fecal shedding distribution. Since we specify a family of fecal shedding distributions, the incidence is computed as an ensemble by repeatedly taking draws from this family of distributions and performing the deconvolution. This ensemble is then summarized for each date with a mean, lower 95% confidence interval (CI) bound, and upper CI bound, which is reported in the `inc` data frame.
+
+- `R`: the daily reproduction number, taken by performing a deconvolution of inferred daily incidence with a generation interval distribution. $\mathcal{R}_t$ is computed in an ensemble by repeatedly taking a draw from the realizations of the inferred incidence and performing the deconvolution with an individual generation interval distribution drawn from the specified family (`dist.gi`). This ensemble is then summarized for each date with a mean, lower CI bound, and upper CI bound. The CI width can be specified by the user in `prm.R`.
```

Comment:
good catch! i thought that's what `EpiEstim::estimate_R()` was doing but i looked at the paper and realized that's not right. i took the idea from your edit above and added a reference to `EpiEstim::estimate_R()`, so that users can look at that documentation if they're interested.